pyccl/tests/test_pkhm.py

```diff
@@ -19,6 +19,24 @@
 PK2D = ccl.Pk2D(cosmo=COSMO, pkfunc=lambda k, a: a / k)
 
 
+def test_hmc_from_string_smoke():
```

Comment:
can you add a quick test that these calculators return the same value for one of their associated methods/functions if initialized from objects or strings?

Reply:
OK

Reply:
I couldn't see these tests @nikfilippas

Reply:
@damonge issue with git, they now show in the line below.

Reply:
They're not qualitative with the string meaning of the word, but I do check now, that the type (!) returned is identical, so they all point to the same class.